run_integration_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "Running integration tests triggered by $TRAVIS_EVENT_TYPE..."
+if [[ $TRAVIS_EVENT_TYPE == 'cron' ]]; then
+  echo 'the integration test command for push will be enabled after env vars are set up for Travis CI'
+#  pytest tests/integration/test_push.py
```

Comment:
We probably shouldn't merge something that references a code path that doesn't exist yet.

Reply:
@kevinrizza the `echo` command serves as a placeholder and the actual command is commented out.

I wanted to make the PRs more separated since they serve as two purposes. And it currently won't break anything and we don't have cron jobs yet.

Do you still want me to put them all in one PR?

Reply:
@jeremylinlin I just don't see why we want to merge this first, given that it doesn't do anything. Pull requests should be singular in purpose, but if your purpose is to add new tests, and those tests require stuff around them to work correctly, they should be in one pr. If you prefer, you could make it two commits in the same pr.